### ISSUE TITLE
Can't verify incoming webhooks

### ISSUE THREAD

**Issue by miken32** [helpscout/helpscout-api-php#272]
* PHP version: 8.0
* SDK version: 3.5.0

**Current behavior**
Signature mismatch: Expected signature is FLX317e8jkhTIjNkvivhXLbAY/E=. No signature was provided.

**Expected behavior**
Signature match

Your API looks for headers `HTTP_X_HELPSCOUT_SIGNATURE` and `X_HELPSCOUT_SIGNATURE`. Neither of these are contained in the request object generated by Laravel using the nyholm/psr7 library.

```none
  -headerNames: array:10 [
    "host" => "host"
    "user-agent" => "user-agent"
    "content-length" => "content-length"
    "accept" => "accept"
    "accept-encoding" => "accept-encoding"
    "content-type" => "content-type"
    "x-forwarded-for" => "x-forwarded-for"
    "x-forwarded-proto" => "x-forwarded-proto"
    "x-helpscout-event" => "x-helpscout-event"
    "x-helpscout-signature" => "x-helpscout-signature"
  ]
```
The code should be updated to look for `x-helpscout-signature` with hyphens in addition to underscores.

**Comment by miken32** [comment]
Same for `x-helpscout-event` of course.

**Comment by bkuhl** [comment]
Thanks for sharing this, I'll submit a PR adding them.

**Comment by bkuhl** [comment]
This is now available in [v3.5.1](https://github.com/helpscout/helpscout-api-php/releases/tag/3.5.1)!  Thanks for bringing this up
